examples/stm32/WolfSSL/.project

```diff
@@ -320,5 +320,10 @@
 			<type>1</type>
 			<locationURI>PARENT-3-PROJECT_LOC/src/import/tls/wolfssl/ctaocrypt/src/wolfcrypt_last.c</locationURI>
 		</link>
+		<link>
+			<name>wolfmath.c</name>
+			<type>1</type>
+			<locationURI>PARENT-3-PROJECT_LOC/src/import/tls/wolfssl/wolfcrypt/src/wolfmath.c</locationURI>
```

Comment:
ok, it is a new file came into wolfSSL on 2016 dec 21, right after v3.10.0-stable